### ISSUE TITLE
Rendering amenity=health_post

### ISSUE THREAD

**Issue by mfrasca** [gravitystorm/openstreetmap-carto#4169]
The need to tag and render lower-than-doctors healthcare facilities was expressed and discussed in both the `tagging` mailing list, and in the OSM Africa Tagging Telegram group.  The proposal reached the tagging mailing list from Claire Halleux, Message-ID: <CACfGQQZbPeH=fTP7jRr+TtGhx4ydtd0HBa3-d86qqSS4Yftsmw@mail.gmail.com>.

Mappers have started to use the tag `amenity=health_post`.

https://taginfo.openstreetmap.org/tags/amenity=health_post

It would be nice to have it rendered.

**Comment by mfrasca** [comment]
without being a graphical designer, I've proposed this idea: it's a shelter, with some medical care.

![image](https://user-images.githubusercontent.com/122869/85430501-cfafd780-b545-11ea-9859-23760d40c320.png)

[health-post.svg.gz](https://github.com/gravitystorm/openstreetmap-carto/files/4820704/health-post.svg.gz)

**Comment by matkoniecz** [comment]
It is used just 321 times worldwide - https://taginfo.openstreetmap.org/tags/amenity=health_post

**Comment by mfrasca** [comment]
this tag comes from a very recent agreement, and it's **already** used 321 times worldwide.  :-)

**Comment by Adamant36** [comment]
> this tag comes from a very recent agreement, and it's already used 321 times worldwide. :-)

From the discussion on the wiki that I participated in, and never got a response to a question about, it doesn't seem there was any agreement about the tag. Just ignoring questions and opposition about the tag isn't "agreement." 

 The 321 uses is extremely small anyway and plus it's only used in a few specific places as it is. There would have to be wider adaptation of the tag then a couple of specific places in Africa and India. Whatever the usage numbers were.

**Comment by pnorman** [comment]
We'd need wider adoption of this tag before adding a unique rendering for it.

**Comment by jeisenbe** [comment]
@mfrasca although this issue is currently closed, we can reopen it later if this tag becomes clearly established as a wide-spread way to map these features. Perhaps sometimes next year?

**Comment by matkoniecz** [comment]
> before adding a unique rendering for it.

or any rendering

**Comment by mfrasca** [comment]
> From the discussion on the wiki that, I participated in, and never received a response to a question about the tag in, it doesn't seem there was any agreement about the tag. Just ignoring questions and opposition about the tag isn't "agreement."

decision forming is different region-to-region.

**Comment by Adamant36** [comment]
> decision forming is different region-to-region.

Obviously. Your requesting something be rendered on a global map though. There has to more involved then just people in your local mapping community discussing and deciding to use it, because everyone will be using if its rendered. So, your "regional" (whatever that means anyway) decision process isn't really that relevant. At least to what to render. Maybe a little, but likely not at all. 

You'll never get the wider adoption of the tag needed for rendering if your hostile toward or ignore people that want to use it but are still on the fence about it. There's been plenty of tags that aren't ultimately rendered even if they are used a ton but only in a specific region, Usually in those cases its better for whoever wants the tag rendered to create a specialized map instead of rendering it on the main site, because again its a global map. Personally, I dont see the tag being widely adopted. Especially with the attitudes seeming to be involved. I'm willing to be proved wrong though.

**Comment by imagico** [comment]
This has been mentioned today in the OSM-Carto map reading workshop.  The tag is now used 1744 times:

https://taginfo.openstreetmap.org/tags/?key=amenity&value=health_post

But there is also [healthcare=nurse](https://wiki.openstreetmap.org/wiki/Tag:healthcare%3Dnurse) - with similar meaning and 1354 uses.  So we quite definitely don't have clear established tagging conventions here.

**Comment by jeisenbe** [comment]
Now used almost 3000 times. Reopening
